COOKBOOK.md

```diff
@@ -0,0 +1,200 @@
+# Cookbook
+
+#### What is the Ion Object Mapper
+
+The Ion Object Mapper is a convenience library built to assist developers when dealing with data that's in the format of [Amazon Ion](https://amzn.github.io/ion-docs/). It is used to easily transform C# classes and objects into Ion format and vice-versa. There are numerous additional settings and features that can be configured for advanced users.
+
+#### Basic Serialization and Deserialization
+
+The mapper (IonSerializer) supports by default the types listed in the [spec](SPEC.md#primitive-type-conversion) under the "Primitive type conversion" section. There are ways to customize the conversion but the default use case adheres to the spec.
```

Comment:
Feel free to disagree but I think spelling out `under the "Primitive type conversion" section` is a bit redundant now that the spec link is pointing right at it (and likewise for the other spec link).

Reply:
I agree with you, but have had many times in the past in documentation in other places where the link breaks for whatever magical reasons so I prefer leaving it for safety.

Reply:
Fair enough.